rebar.config

```diff
@@ -17,9 +17,19 @@
 
 {deps, [
         {jsx, "2.8.0"},
-        {lhttpc, "1.4.0"}
+        {lhttpc, "1.4.0"},
+        {eini, ".*",
+         {git, "https://github.com/accense/eini.git",
+          {branch, "master"}}}
```

Comment:
I think you should use a tag or a hash, not "master".

Having a non-Hex dependency makes publishing to Hex problematic.